CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.11.0]
+### Removed
+- **BREAKING:** Remove `wallet_swapAsset` ([#289](https://github.com/MetaMask/api-specs/pull/289))
```

Comment:
I assume this is breaking? I don't quite know how it works when what we are releasing is not JavaScript code but a JSON file. But if consumers are using this now and can't use it anymore it seems like a breaking change to me.